en_US.ini

```diff
@@ -598,10 +598,10 @@ Force real clock sync (slower, less lag) = Force real clock sync (slower, less l
 General = General
 Help the PPSSPP team = Help the PPSSPP team
 Host (bugs, less lag) = Host (bugs, less lag)
-I/O on thread (experimental) = I/O on thread (experimental)
+I/O on thread (experimental) = I/O on thread
 IO timing method = I/O timing method
 MMDDYYYY = MMDDYYYY
-Multithreaded (experimental) = Multithreaded (experimental)
+Multithreaded (experimental) = Multithreaded (glitches)
```

Comment:
Is there really any games that glitch instead of causing to them a game breaking bug? I haven't encountered one that was either working fine or not working at all. (Though I have a feeling that it does something weird on Me & My Katamari, I could be wrong)